### PR TITLE
Rename examples cmd and add checks

### PR DESCRIPTION
**Pull request by fpliger**
Minor PR to improve developers onboarding. Adds checks for node and npm versions and rename `example` to `examples`

### CODE REVIEW

**Review by pww217 — APPROVED** [review]
Should be fine but if CI fails when it merges, I'll investigate.

File: pyscriptjs/Makefile [hunk 1]
```diff
@@ -10,8 +10,25 @@ CONDA_EXE := conda
 CONDA_ENV ?= ./env
 env := $(CONDA_ENV)
 conda_run := $(CONDA_EXE) run -p $(env)
+GOOD_NODE_VER := 14
+GOOD_NPM_VER := 6
+NODE_VER    := $(shell node -v | cut -d. -f1 | sed 's/^v\(.*\)/\1/')
+NPM_VER     := $(shell npm -v | cut -d. -f1)
+
+GOOD_NODE   := $(shell if [ $(NODE_VER) -ge $(GOOD_NODE_VER) ]; then echo true; else echo false; fi)
+GOOD_NPM    := $(shell if [ $(NPM_VER) -ge $(GOOD_NPM_VER) ]; then echo true; else echo false; fi)
```
Comment (marimeireles) [inline pyscriptjs/Makefile:19]: Is there an advantage of using `-ge`? Or is it equivalent to `>=`? I feel like the latter is better.

File: pyscriptjs/Makefile [hunk 2]
```diff
@@ -10,8 +10,25 @@ CONDA_EXE := conda
...
+
+.PHONY: check-node
```
Comment (marimeireles) [inline pyscriptjs/Makefile:21]: This is a question because I don't know: why don't we use `.PHONY` in all commands?

**Review by marimeireles — COMMENTED** [review]
Great! :)
Just a nit picking and a question.

File: pyscriptjs/Makefile (hunk 1 above)
Comment (fpliger) [reply pyscriptjs/Makefile:19]: I don't think there's an advantage. Just always used (and seen) this way... I think double parenthesis can be used as well for performing numeric comparison but may be wrong here...

File: pyscriptjs/Makefile (hunk 2 above)
Comment (fpliger) [reply pyscriptjs/Makefile:21]: Here's a bit more on what a `.PHONY` target is. Honestly, I've never really been super strict about it... that's probably why 😄  . We could actually (most of the cmds at least), but I wouldn't do it in this PR. Are you ok to merge as is?

Comment (marimeireles) [reply pyscriptjs/Makefile:21]: Let's do it :)

**Review by marimeireles — APPROVED** [review]

**Pull request merged**